### ISSUE TITLE
Print Screen button seems to be broken in 0.5.1

### ISSUE THREAD

**Issue by Kevin43** [moonlight-stream/moonlight-qt#85]
**Describe the bug**
Ever since before I updated to the newest release(0.5.1), I could use the print screen button normally, but now it seems like it isn't captured. 

**Steps to reproduce**
1. Connect to your desktop via mstsc.
2. Open this site http://www.keyboardtester.com/
3. Press the print screen key, which won't show.

**Screenshots**
If applicable, add screenshots to help explain your problem. If the issue is related to video glitching or poor quality, please include screenshots.

**Moonlight settings (please complete the following information)**
- Have any settings been adjusted from defaults?
- If so, which settings have been changed?
- Does the problem still occur after reverting settings back to default?

**Client PC details (please complete the following information)**
 - OS: Windows 10 Enterprise 1709
 - Moonlight Version: [0.5.1]

**Logs (please attach)**
Will reply with files, since I have it on another computer.

**Comment by Kevin43** [comment]
[Moonlight-1537449700.log](https://github.com/moonlight-stream/moonlight-qt/files/2401627/Moonlight-1537449700.log)

**Comment by cgutman** [comment]
This is actually a regression in GeForce Experience 3.15. I can reproduce the issue on my GFE 3.15 test machine but not on the one running GFE 3.14.1.

**Comment by Kevin43** [comment]
Ah I see, does this mean the problem is out of your hands, until Nvidia fixes it? If so, that's a shame.

**Comment by cgutman** [comment]
Yes, but you can downgrade to GFE 3.14.1 if this is a showstopper for you.
